res/cmake/dep/highfive.cmake

```diff
@@ -29,6 +29,8 @@ if(HighFive)
   )
   set(HIGHFIVE_USE_BOOST OFF)
   set(HIGHFIVE_BUILD_DOCS OFF) # conflicts with phare doc target
+  set(HIGHFIVE_EXAMPLES OFF)
+  set(HIGHFIVE_PARALLEL_HDF5 ON) # causes failure if cannot occure
```

Comment:
I don't understand the comment

Reply:
maybe unnecessary, if it's ON, and installed hdf5 is serial, an error is thrown

Reply:
an error is thrown at compile time (link time) ? because what? mpi-io stuff?

Reply:
I've removed it, if we want to consider it another time we can

Reply:
ok make an issue if you think it's worth it I have no strong opinion as I don't (yet) completely understand the point